resources/examples/manager/BasicAssetLibrary/tests/fixtures.py

```diff
@@ -0,0 +1,105 @@
+#
+#   Copyright 2013-2022 The Foundry Visionmongers Ltd
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+"""
+Manager test harness fixtures for the apiComplianceSuite
+"""
+
+import json
+import os
+
+
+from openassetio.constants import kField_EntityReferencesMatchPrefix
+
+#
+# Test library
+#
+
+test_library_path = os.path.join(os.path.dirname(__file__), "resources", "library_apiComplianceSuite.json")
+test_libray = {}
+
+with open(test_library_path, "r", encoding="utf-8") as file:
+    test_library = json.load(file)
+
+#
+# Constants
+#
+
+identifier = "org.openassetio.examples.manager.bal"
+
+valid_ref = "bal:///references/can/contain/🐠"
+non_ref = "not a Ŕeference"
+
+an_existing_ref = next(iter(test_library["entities"].keys()))
+
+
+fixtures = {
+    "identifier": identifier,
+    "settings": {
+        "library_path": test_library_path
+    },
+    "shared": {
+        "a_valid_entity_reference": valid_ref,
+        "a_non_entity_reference": non_ref,
+    },
+    "Test_identifier": {
+        "test_matches_fixture": {
+            "identifier": identifier
+        }
+    },
+    "Test_displayName": {
+        "test_matches_fixture": {
+            "display_name": "Basic Asset Library 📖"
+        }
+    },
+    "Test_info": {
+        "test_matches_fixture": {
+            "info" : {
+                kField_EntityReferencesMatchPrefix: "bal:///"
+            }
+        }
+    },
+    "Test_setSettings": {
+        "shared": {
+            "some_valid_settings": {"library_path": "/dev/null"},
+            "some_settings_with_unknown_keys": {"cat": True}
```

Comment:
I wondered where on the fence that sat too. We say in the API contract (or will do, if we don't already) that unknown keys should cause an error.  So I guess if we test value types, we need to state the behaviour then (so that it is in API Contract territory rather than local business logic)?

Reply:
Maybe the language should be "invalid entries" rather than "invalid keys"?

In the docs `setSettings` is currently blank for `Manager` and says "Todo: Document settings mechanism" for `ManagerInterface`.